src/ECER.Tests/ECER.Tests.csproj

```diff
@@ -26,6 +26,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\ECER.Clients.RegistryPortal\ECER.Clients.RegistryPortal.Server\ECER.Clients.RegistryPortal.Server.csproj" />
+    <ProjectReference Include="..\ECER.Managers.Registry.Contract\ECER.Managers.Registry.Contract.csproj" />
```

Comment:
I don't think this is required  as it should be referenced by the portal

Reply:
I need this for Unit tests to test the validation engine. see `ApplicationSubmissionValidationEngineTests.cs`